client/my-sites/site-settings/cloudflare.js

```diff
@@ -23,7 +23,7 @@ const Cloudflare = () => {
 	const hasJetpackCDN = useSelector( ( state ) =>
 		siteHasFeature( state, siteId, WPCOM_FEATURES_CDN )
 	);
-	const upgradeLink = `/plans/${ siteSlug }?customerType=business`;
+	const upgradeLink = `/checkout/${ siteSlug }/pro`;
```

Comment:
UpsellNudge uses this link as default if we do not set a `href` attribute. Would it make sense to remove it entirely and rely on default behavior?

Reply:
@danielbachhuber, thank you for providing the details. Relying on the default gives some benefits e.g., the upsell logic becomes more centralized, but the way with link also works fine.